run_mypy.py

```diff
@@ -45,6 +46,7 @@
     'mesonbuild/mtest.py',
     'mesonbuild/optinterpreter.py',
     'mesonbuild/programs.py',
+    'mesonbuild/wrap/wraptool.py',
```

Comment:
Don't list this in "specific files", it is already listed above in "fully typed modules".

This is the cause of the CI failure: 
```
mesonbuild/wrap/wraptool.py: error: Duplicate module named "mesonbuild.wrap.wraptool" (also at "mesonbuild/wrap/wraptool.py")
```